docs/en/reference/pipeline-integrations/user-guide/sa-integrations-user-guide.md

```diff
@@ -0,0 +1,509 @@
+---
+layout: default
+title: Integrations User Guide
+pagename: integrations-user-guide
+lang: en
+---
+
+<!-- omit in toc -->
+# Integrations user guide
+
+In this guide:
+- [Introduction](#introduction)
+- [Getting started with {% include product %} Desktop](#getting-started-with--include-product--desktop)
+  - [Installation of Desktop](#installation-of-desktop)
+  - [Logging in](#logging-in)
+    - [Authentication and browser integration](#authentication-and-browser-integration)
+  - [The system tray](#the-system-tray)
+  - [The project browser](#the-project-browser)
+  - [The command launcher](#the-command-launcher)
+  - [The console](#the-console)
+- [Launching applications](#launching-applications)
+- [The {% include product %} Panel](#the--include-product--panel)
```

Comment:
This link is having the same issue, maybe it's the variable or the extra dashes?

Reply:
Some solution here.